Makefile

```diff
@@ -142,10 +142,13 @@ lint: ## Run linters. Use make install-linters first.
 	# The govet version in golangci-lint is out of date and has spurious warnings, run it separately
 	go vet -all ./...
 
+check-coin: newcoin ## Check that make newcoin succeeds and no files are changed.
+	if [ "$(shell git diff ./cmd | wc -l | tr -d ' ')" != "0" ] ; then echo 'Changes detected after make newcoin' ; exit 2 ; fi
```

Comment:
there should be no changes at all, not just in `./cmd` (the `./src/params/` folder also has a templated file)